psi4/src/psi4/libmoinfo/moinfo.h

```diff
@@ -74,7 +72,7 @@ class MOInfo : public MOInfoBase {
         const MOInfo* moinfo;
 
        public:
-        typedef std::bitset<size_det> bitdet;
+        typedef std::bitset<2048> bitdet;  // adjust based on "size det"?
```

Comment:
Just to double check: was `size_det` used only here? It might be a good idea to make this a static constant expression in the class `SlaterDeterminant`.

Reply:
Yes, this was the only hit for `size_det` in the entire repo. I have also searched for 2048 to check for things assuming this size, and found nothing. So having it "inline" like this seems like the most straightforward solution.